scalableminds/webknossos
# Color design of dataset gallery

### URL of deployed dev instance (used for testing):
- [test here](https://colordesignofdatasetgallery.webknossos.xyz)

### Steps to test:
- open the gallery dataset view and resize the window
- logout and reload the dataset view -> the buttons for skeleton and volume tracing should now be transparent 


### Issues:
- fixes #2606

------
